contrib/linting.md

```diff
@@ -0,0 +1,44 @@
+# Keeping our code style aligned with linting
+
+We're using linters for a couple of reasons:
+- We've experienced a lot of reviews to focus on the stying of code rather than its design and architecture. In particular, this has the potential to be a real turn-off for a newcomer.
+- Writing code in a shared style makes way easier to read across files.
+- As with any static analysis of code, linting has the potential to find bugs before our users do.
+
+## Setup your editor to format on save
+
+To help yourself keep your code passing linters at all times, it's highly recommended to setup your editor to help you.
+
+### Visual Studio Code
+
+- Install and enable the ["eslint" extension for VS Code](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
```

Comment:
We could add it to `extensions.json` (https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions)

Reply:
I believe I already did that 🙂